src/components/footer/www/footer.jsx

```diff
@@ -219,12 +219,19 @@ const Footer = props => (
 
 Footer.propTypes = {
     intl: intlShape.isRequired,
-    scratchWikiLink: PropTypes.string
+    scratchWikiLink: PropTypes.string,
+    isFullScreen: PropTypes.bool
```

Comment:
I wonder if a name like "projectIsFullScreen" would make more sense here, since the footer itself is not full screen